app/scripts/archive/views/archive.html

```diff
@@ -0,0 +1,28 @@
+<h1>Archive</h1>
+
+{{ status }}
+
+<table border="1" pading="10px">
+
+    <th>ID</th>
+    <th width="200px">Titles</th>
+    <th width="300px">Authors</th>
+    <th width="200px">State</th>
+
+    <tr ng-repeat="article in articles | orderBy:'id'">
+        <td> <a href="#/article/{{article.id}}">{{ article.id }}</a> </td>
+        <td>
+            <div ng-repeat="title in article.titles">
+                {{ title.title }}
+            </div>
+        </td>
+        <td>
+            <div ng-repeat="author in getAuthorsInfo(users, article.authors)">
+                <a href="#/user/{{author.id}}">{{ author.email }} </a>
+            </div>
+        </td>
+        <td>
+            {{ getStateInfo(states, article.state ).name }}
```

Comment:
Looks like a hack, this would ideally be an object called `user` or something like that.